scrc/preprocessors/extractors/bge_reference_extractor.py

```diff
@@ -57,8 +57,8 @@ def select_df(self, engine: str, spider: str) -> str:
     def save_data_to_database(self, df: pd.DataFrame, engine: Engine):
         """Instead of saving data into database, references get written into a text file"""
         self.logger.info("save data in progress")
-        processed_file_path = ROOT_DIR / 'data' / 'datasets' / "bge_references_found.txt"
-        not_processed_file_path = ROOT_DIR / 'data' / 'datasets' / "bge_not_extracted.txt"
+        processed_file_path = ROOT_DIR / 'data' / 'datasets' / 'criticality_prediction' / "bge_references_found.txt"
```

Comment:
please use the paths that are set already in the abstract classes using the config instead of constructing them yourself.

Reply:
I will change this.